src/lib/offchain.ts

```diff
@@ -1,21 +1,36 @@
 import { type Addressable, type Log, EventFragment, keccak256 } from 'ethers'
 
-import { type CCIPMessage, type CCIPRequest, defaultAbiCoder } from './types.js'
+import {
+  type CCIPMessage,
+  type CCIPRequest,
+  defaultAbiCoder,
+  parseSourceTokenData,
+} from './types.js'
 import { networkInfo } from './utils.js'
 
 const USDC_EVENT = EventFragment.from('MessageSent(bytes message)')
 const TRANSFER_EVENT = EventFragment.from('Transfer(address from, address to, uint256 value)')
+export const LBTC_EVENT = EventFragment.from(
+  'DepositToBridge(address fromAddress, bytes32 toAddress, bytes32 payloadHash, bytes payload)',
+)
 
 const CIRCLE_API_URL = {
   mainnet: 'https://iris-api.circle.com/v1',
   testnet: 'https://iris-api-sandbox.circle.com/v1',
 }
+const LOMBARD_API_URL = {
+  mainnet: 'https://bridge-manager.lombard.finance',
+  testnet: 'https://bridge-manager.staging.lombard.finance',
+}
 
 type AttestationResponse =
   | { error: 'string' }
   | { status: 'pending_confirmations' }
   | { status: 'complete'; attestation: string }
 
+type LombardAttestation = { status: string; message_hash: string; attestation: string }
```

Comment:
Nit: I see you compare `status` here with some constant. You don't need to extract that const in a global var, but if you can get a small list of possible status, it'd be nice to hardcode them as a union of string literals here, as it'll aid the constant comparison down there.
```suggestion
type LombardAttestation = { status: string; message_hash: string; attestation: string }
```

Reply:
done